EIPS/eip-4750.md

```diff
@@ -133,6 +133,17 @@ Alternative logic for `RETF` in the top frame could be to allow it during code v
 
 This has been superseded with the validation rule of top frame (0th code section) being non-returning (non-returning sections introduced in a separate EIP), because validating non-returning status of functions is valuable by itself for other reasons. Therefore all considerations of runtime behavior of `RETF` in the top frame were obsoleted.
 
+### "Minimal" function type
+
```

Comment:
you don't want to convert this EIP from stagnant to draft or something?

Reply:
I do want.

Reply:
It should go back to Review, but I'm also doing it in https://github.com/ethereum/EIPs/pull/9386